registry/endpoint.go

```diff
@@ -189,29 +194,14 @@ func isSecure(hostname string, insecureRegistries []string) (bool, error) {
 		// So, len(addrs) == 0 and we're not aborting.
 	}
 
-	for _, r := range insecureRegistries {
-		if hostname == r {
-			// hostname matches insecure registry
-			return false, nil
-		}
-
-		// Try CIDR notation only if addrs has any elements, i.e. if `host`'s IP could be determined.
```

Comment:
@dkjer I'd like to keep as many relevant comments as possible (like this one), for ease of understanding.